.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: scala
+scala:
+  - 2.12.3
+jdk:
+  - oraclejdk8
+script: sbt clean coverage test
+deploy:
+  provider: script
+  script: src/bin/publish_release.sh
+  on:
+    tags: true
```

Comment:
While it would be nice to deploy releases, this script doesn't appear to be included in your branch, and thus will just break whenever someone creates a git tag in the future.